plugins/inputs/modbus/modbus_test.go

```diff
@@ -229,6 +262,8 @@ func TestGetRTUdataReadInputRegister(t *testing.T) {
 }
 
 func TestGetRTUdataWriteSingleCoil(t *testing.T) {
+		// Note: Address may be offset by 1
```

Comment:
Run `make fmt`

Reply:
Thanks, yes, I noticed that extra tab later.

Ran `go fmt` on all files in Modbus Plugin.